src/modules/CSADigitizer/README.md

```diff
@@ -0,0 +1,41 @@
+# CSADigitizer
+**Maintainer**: 
+**Status**: 
+**Input**: PixelCharge  
+**Output**: PixelHit  
+
+### Description
+Digitization module which translates the collected charges into a digitized signal, emulating a charge sensitive amplifier with Krummenacher feedback.
+
```

Comment:
Please extend this description with the things you have presented in your talks and also give the reference to the transfer fucntion you are using and the functions themselves (can be done with backtick-dollar:

`$ test_0$`

Reply:
(works in GitLab)